packages/react-theme-provider/src/callOrReturn.ts

```diff
@@ -0,0 +1,9 @@
+// type VariantClassesHook = <TState>(state: TState) =>
+/**
+ * Calls a function with the argument, or returns the given object.
+ * @param objOrFunc - Function or object.
+ * @param argument - Argument to pass if a function is provided.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const callOrReturn = (objOrFunc: any, argument: any) =>
```

Comment:
Is this something that should be in `react-theme-provider`? Seems like a more general utility.

Reply:
it's not exported by theme-provider and is only used there.

Reply:
But even then, moving it would not be a breaking change and we could do later if we find more use cases.